oaknational/Oak-Web-Application
# LESQ-886: New content banner

## Description

Music year: 1951

- Creates new content banner in OWA using components from component lib
- Adds content banner on unit listing, lesson listing and lesson overview pages
- Links to correct programme or unit listing page as prescribed in this doc: https://docs.google.com/spreadsheets/d/1pE0VtBgSsokQbCrhIhNJrhHfZrnS0U7xy57zsqexmmw/edit#gid=0 

## Issue(s)

Fixes #LESQ-886

## How to test

1. Go to https://deploy-preview-2511--oak-web-application.netlify.thenational.academy
2. Click on any subject with new content
3. You should see banner should be rendered on unit listing, lesson listing and lesson overview pages of all subjects apart from Maths, which is lesson listing and lesson overview
4. Go to resource button should navigate to the new resource unit listing page, unless their are programme factors then programme listing page

## Screenshots

How it should look now: 
<img width="980" alt="Screenshot 2024-06-18 at 10 32 02" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/c25b8129-f559-45ea-9359-06da1b0fcb1b">

<img width="1206" alt="Screenshot 2024-06-18 at 10 33 10" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/025fde4f-1b1a-412d-863a-9a25882136b8">

<img width="381" alt="Screenshot 2024-06-18 at 10 37 50" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/33eabd4a-293a-42e8-9cd3-978893b3e6b6">

## Checklist
